src/Parser/Shaman/Elemental/Modules/Talents/ElementalBlast.js

```diff
@@ -14,6 +14,26 @@ class ElementalBlast extends Analyzer {
     combatants: Combatants,
   };
 
+  currentBuffAmount=0;
+  lastFreshApply=0;
+  resultDuration=0;
+
+  on_removebuff(event) {
+    if (event.ability.guid === SPELLS.ELEMENTAL_BLAST_CRIT.id ||event.ability.guid === SPELLS.ELEMENTAL_BLAST_MASTERY.id ||event.ability.guid === SPELLS.ELEMENTAL_BLAST_HASTE.id){
+      this.currentBuffAmount--;
+      if (this.currentBuffAmount===0)
```

Comment:
Please use braces even around single-line statements. So in this case, 
```jsx
if (this.currentBuffAmount === 0) {
   this.resultDuration += event.timestamp - this.lastFreshApply;
}
```
Same with the following function.